common/argo/base/vars.yaml

```diff
@@ -36,6 +36,30 @@ artifactRepository:
     keyFormat:
       required: true
       default: artifacts/{{workflow.namespace}}/{{workflow.name}}/{{pod.name}}
+  gcs:
+    # Name of bucket, example: my-bucket
+    bucket:
+      required: true
+      default: <bucket-name>
+    # Endpoint for S3 compatible storage
+    # Supported provider endpoints:
+    #   AWS: s3.amazonaws.com
+    #   GCS: storage.googleapis.com
+    #   Minio: my-minio-endpoint.default:9000
+    endpoint:
+      required: true
+      default: storage.googleapis.com
+    # Change to true if endpoint does NOT support HTTPS
+    insecure:
+      required: true
+      default: false
+    serviceAccountKey:
+      required: true
+      default: '| <key.json-file-data>'
```

Comment:
Not sure if this is possible but can this default be shown like this?
```yaml
default: |
  <key.json>
```

Reply:
Done